src/bitmap/immutable.rs

```diff
@@ -239,7 +239,12 @@ impl Bitmap {
     pub fn make_mut(self) -> MutableBitmap {
         match self.into_mut() {
             Either::Left(data) => {
-                MutableBitmap::from_vec(data.bytes.as_ref().to_vec(), data.length)
+                if data.offset > 0 {
+                    // we have to recreate the bytes because a MutableBitmap does not have an `offset`.
+                    data.iter().collect()
```

Comment:
I think we can improve this further by using `iter_chunk`, which will iterate in chunks of `u64`. We have a function somewhere to create a `MutableBitmap` from chunks.

Reply:
I tried this, but that seems to not include the `remainder`

```rust
                 let chunks = BitChunks::<u64>::new(data.bytes.as_ref(), data.offset, data.length);
                    chunk_iter_to_vec(chunks)
```